packages/@aws-cdk/aws-apigateway/lib/stage.ts

```diff
@@ -13,6 +15,22 @@ export interface StageOptions extends MethodDeploymentOptions {
    */
   readonly stageName?: string;
 
+  /**
+   * The CloudWatch Logs log group.
+   *
+   * @default - No destination
+   */
+  readonly accessLogDestination?: IAccessLogDestination;
+
+  /**
+   * A single line format of access logs of data, as specified by selected $content variables.
+   * https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-mapping-template-reference.html#context-variable-reference
+   * The format must include at least `$context.requestId`.
+   *
+   * @default - No format
```

Comment:
When accessLogDestination is specified, we should find a reasonable default for this property. The CLF format seems like a reasonable default, yes?